src/views/payment-sheet-views/card-view.js

```diff
@@ -203,7 +203,7 @@ CardView.prototype._generateHostedFieldsOptions = function () {
   if (!overrides) { return options; }
 
   if (overrides.fields) {
-    if (overrides.fields.cvv && overrides.fields.cvv.placeholder) {
```

Comment:
Shouldn't this be `overrides.fields.cvv.placeholder !== undefined`?

I think any string coercion you're seeing is related to the demo app.